### PR TITLE
[NU-1648] Fix ad hoc tests with schemas containing nested fields

### PR DESCRIPTION
**Pull request by jedrz**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/TestingParametersSupport.scala [hunk 1]
```diff
@@ -10,20 +10,39 @@ object TestingParametersSupport {
 
   // Used to un-flat map with concat name eg. { a.b -> _ } => { a -> { b -> _ } }. Reverse kinda joinWithDelimiter
   def unflattenParameters(flatMap: Map[ParameterName, AnyRef]): Map[String, AnyRef] = {
+    unflattenParametersInternal(flatMap.map { case (ParameterName(name), value) => name -> value })
```
Comment (philemone) [inline utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/TestingParametersSupport.scala:13]: It probably was me who named it `flatMap` but now I think that it is unfortunate name for scala variable :face_in_clouds: Maybe `flatParameters` / `flatParams` or something like this?

**Review by philemone — APPROVED** [review]
nice, LGTM :+1:

File: utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/TestingParametersSupport.scala (hunk 1 above)
Comment (jedrz) [reply utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/TestingParametersSupport.scala]: You're right. It's a better name.

**Review by coutoPL — APPROVED** [review]
LGTM

**Pull request merged**